maintainers/maintainer-list.nix

```diff
@@ -119,6 +119,12 @@
     githubId = 12578560;
     name = "Quinn Bohner";
   };
+  aacebedo = {
```

Comment:
This file should be in a separate commit.

Also put this in alphabetical order.

Reply:
corrected